lib/schemas/api.js

```diff
@@ -1,9 +1,3 @@
 const joi = require('joi')
 
-module.exports = joi.object().keys({
-  controllers: joi.object().required(),
-  models: joi.object().required(),
-  policies: joi.object().required(),
-  services: joi.object().required(),
-  config: joi.object().optional()
-})
+module.exports = joi.object().required()
```

Comment:
controllers, models, policies, services and config are required by Trails and should stay like this, if you want to allow more keys take a look at https://github.com/hapijs/joi/blob/v9.0.0-5/API.md#objectunknownallow to allow any keys but required the ones we need